scvi/_settings.py

```diff
@@ -1,5 +1,20 @@
 import logging
+from typing import Union
 
+scvi_logger = logging.getLogger("scvi")
 
-def set_global_verbosity(level: str):
-    logging.getLogger("scvi").setLevel(level)
+
+def set_verbosity(level: Union[str, int]):
+    """Sets logging configuration for scvi based on chosen level of verbosity."""
```

Comment:
Maybe explain what the code does (only add a console handler if there is not already one).
And that it is a console handler to print logs (mostly) in notebooks.

Reply:
Also what if you configure the `scvi_logger` to be debug and the stream_handler to be `info`. This way the user can eventually add a debug handler without changing scvi verbosity (I have the feeling it's cleaner this way, otherwise the handler level is useless).

Reply:
I updatded the docstring. On  your second comment: I don't think we want to send the debug logs to the root logger if the user has specified a info level.